imports/plugins/custom/faq/client/stylesheets/index.less

```diff
@@ -0,0 +1,98 @@
+.faq-container {
+  padding: 30px 0px 15px 0;
+  display: flex;
+  margin: 0 auto;
+  width: 80%;
+}
+
+.faq-content {
+  width: 100%;
+
+  h2 {
+    font-size: 32px;
+    font-style: normal;
+    font-weight: bold;
+    text-transform: uppercase;
+    line-height: 1.5;
+    color: #000000;
```

Comment:
I believe hex codes should be stored in variables and referenced using their variable names. It's not really important but you should try storing the hex codes in variables and reference them with their names if time permits.